hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioThread.java

```diff
@@ -144,7 +144,7 @@ public long priorityFramesTransceived() {
     }
 
     public long handleCount() {
```

Comment:
The method name would also deserve renaming.

Reply:
Good spotting. It is already in one of my other pr's